packages/codemirror/src/DOTEditor.ts

```diff
@@ -0,0 +1,20 @@
+import { Editor } from "./Editor";
+
+export class DOTEditor extends Editor {
+    options(): any {
+        return {
+            ...super.options(),
+            mode: "text/x-dot"
+        };
+    }
+
+    dot(): string;
+    dot(_: string): this;
+    dot(_?: string): string | this {
```

Comment:
Maybe we should solely rely on `.text()` when adding support for a new language? I realize the other languages do this same sort of thing, but is it really necessary?

Reply:
I did have the same thought, but went with the flow.
We could add the "text" method to the base class if it doesn't exist already?